builder/src/lib.rs

```diff
@@ -209,6 +214,10 @@ pub fn build_and_sign_image(fmc: &FwId, app: &FwId, opts: ImageOptions) -> anyho
         vendor_config: opts.vendor_config,
         owner_config: opts.owner_config,
     })?;
+    Ok(image)
+}
+
+pub fn sign_image(image: &ImageBundle) -> anyhow::Result<Vec<u8>> {
```

Comment:
I don't think this function is named correctly; I don't think it's actually doing any signing...

Reply:
Updated the function name.